lean4-infoview/src/infoview/interactiveCode.tsx

```diff
@@ -41,15 +41,15 @@ interface TypePopupContentsProps {
 
 function renderCodeBlock(lang: string, code: string) : string {
   // todo: render Lean code blocks using the lean syntax.json
-  return `<div class="font-code tl pre-wrap">${code}</div>`
+  return `<div class="code-hover-contents font-code tl pre-wrap">${code}</div>`
```

Comment:
I assume we want this to trigger the various `hover-contents` selectors from the Monaco CSS? /CC @lovettchris

Reply:
this is what vscode is producing in the editor, and I do see it using `code-hover-contents` around the top code block containing `Point : Type`, but not around the lean example code block containing `let pt := ...` which is weird.

```html
<div class="monaco-hover-content" style="overflow: hidden; font-size: 14px; line-height: 1.35714; max-height: 250px; max-width: 500px;"><div class="hover-row markdown-hover"><div class="hover-contents code-hover-contents"><div><div data-code="id#21"><span style="font-family: Consolas, &quot;Courier New&quot;, monospace; font-weight: normal; font-size: 14px; font-feature-settings: &quot;liga&quot; 0, &quot;calt&quot; 0; line-height: 19px; letter-spacing: 0px;"><div class="monaco-tokenized-source"><span class="mtk1">Point : Type</span></div></span></div><hr>
<p>This is a <strong>cool</strong> function, it can:</p><ul>
<li>slice bread</li>
<li>make toast</li>
<li>spread marmalade</li>
<li>set a timer</li>
</ul>
<p>Example usage:</p><div data-code="id#22"><span style="font-family: Consolas, &quot;Courier New&quot;, monospace; font-weight: normal; font-size: 14px; font-feature-settings: &quot;liga&quot; 0, &quot;calt&quot; 0; line-height: 19px; letter-spacing: 0px;"><div class="monaco-tokenized-source"><span class="mtk5">let</span><span class="mtk1"> pt := { x : </span><span class="mtk6">1</span><span class="mtk1">, y := </span><span class="mtk6">2</span><span class="mtk1"> }</span></div></span></div><p>See code at <a data-href="http://github.com/leanprover/lean4" href="" title="http://github.com/leanprover/lean4">Foo.lean</a>.</p></div></div></div></div>
```

Reply:
and indeed your version now seems a little too compressed:

![image](https://user-images.githubusercontent.com/18707114/178060707-3177ad8b-9c67-49f5-b9d7-9405929399ab.png)

Reply:
And paragraphs in a list seem quite crowded now:

![image](https://user-images.githubusercontent.com/18707114/178060840-c645bbd4-c4d3-4fa1-87d6-d4ddce436b86.png)

Reply:
and the backtick blocks are broken - no longer monospaced.

Reply:
I added the proper line height calculation. It should hopefully look better. Could you have a look?

Reply:
It's still not identical, but it is much better:
![image](https://user-images.githubusercontent.com/18707114/178813279-99bf7b79-2dab-4fba-baee-b81131e372b2.png)

and
![image](https://user-images.githubusercontent.com/18707114/178813292-073dc6f9-bca2-49f0-b143-6f1cc7d30b64.png)